devops

```diff
@@ -56,6 +56,16 @@ case $COMMAND in
         exit
         ;;
 
+     ci)
+        cp -n ./.env.example ./.env
```

Comment:
würde direkt in der docker-compose.ci alle notwendigen env Variablen definieren.